src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -0,0 +1,341 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class AttributeTests_Embedded : CSharpTestBase
+    {
+        [Fact]
+        public void ReferencingEmbeddedAttributesFromADifferentAssemblyFails_Internal()
```

Comment:
Yes. With the same name in `AttributeTests.vb`

---
In reply to: [115875187](https://github.com/dotnet/roslyn/pull/18715#discussion_r115875187) [](ancestors = 115875187)